functors_task_1/student/include/impl.h

```diff
@@ -16,16 +25,18 @@ struct BeerOrganizer
  *
  * @note Only Corona and HoeGaarden are expensive
  */
-bool isExpensiveBeer(/**???*/)
+bool isExpensiveBeer(BeerBrand brand)
 {
+    return brand == BeerBrand::Corona || brand == BeerBrand::HoeGaarden;
 }
 
 /**
  * @todo Implement lambda beer country equality comparator
  * @return true if beer county is the same, false otherwise
  */
-auto sameCountry = [](/**???*/)
+auto sameCountry = [](BeerBrand b1, BeerBrand b2)
```

Comment:
_Prefer making params const by default, as modifying function input params is a bad pratice_